Session.php

```diff
@@ -61,6 +61,11 @@ public function init()
     {
         parent::init();
         $this->db = Instance::ensure($this->db, Connection::className());
+        if (!$this->getIsActive()) {
+            $this->db
+                ->getCollection($this->sessionCollection)
+                ->createIndex('expire', ['expireAfterSeconds' => 0]);
```

Comment:
This does not change much.
Index is not supposed to be created automatically. It should be created by migration script and not belong here.
That is why back at #16 I said this should be an option like `useTtl`, which can be enabled at will.